### PR TITLE
fixes for correct waits and warning about k8s version

### LINKED ISSUE #27: Fix noobaa installation script
The problem lies in the lines with `sleep 15` like here https://github.com/IBM/dataset-lifecycle-framework/blob/10f3be95913aa9624245c9c48f750a4d7d9dbfc8/examples/noobaa/noobaa_install.sh#L28

We should wait until the objects are ready and not 15 seconds

### PR DESCRIPTION
**Pull request by YiannisGkoufas**
Fixes #27 #12

### CODE REVIEW

File: examples/noobaa/noobaa_install.sh [hunk 1]
```diff
@@ -2,8 +2,42 @@
...
+      echo "Waiting for the 2 noobaa pods to be ready"
+      sleep 5
+    done
+}
+
+function wait_for_pods_running(){
+	kubectl wait --for=condition=ready pods -l app=noobaa > /dev/null 2>&1
 }
 
 function install_noobaa() {
```
Comment (christian-pinto) [inline examples/noobaa/noobaa_install.sh:44]: Indentation is weird for this function. Please fix.

File: examples/noobaa/noobaa_install.sh [hunk 2]
```diff
@@ -2,8 +2,42 @@
...
+        echo "Noobaa pods ready!"
+        break
+      fi
+      echo "Waiting for the 2 noobaa pods to be ready"
+      sleep 5
+    done
+}
+
+function wait_for_pods_running(){
+	kubectl wait --for=condition=ready pods -l app=noobaa > /dev/null 2>&1
```
Comment (christian-pinto) [inline examples/noobaa/noobaa_install.sh:41]: This is also indented differently from rest of the code.

**Comment by YiannisGkoufas** [comment]
@christian-pinto should be fine now

**Comment by christian-pinto** [comment]
It all looks good now, thanks!

**Pull request merged**